mypy/fastparse.py

```diff
@@ -2249,7 +2249,7 @@ def visit_member_expr(self, e: MemberExpr) -> None:
 
 
 class FindYield(TraverserVisitor):
-    """Check if an AST contains yields or yield froms."""
+    """Check if an AST contains yields or yield forms."""
```

Comment:
I think this refers to "yield from" (https://docs.python.org/3/whatsnew/3.3.html#pep-380), so it's written as intended?

Reply:
It seems like "from" is the correct terminology as per PEP 380 (https://docs.python.org/3/whatsnew/3.3.html#pep-380), so the original phrasing is accurate. The update might not be necessary.